.github/workflows/ci.yml

```diff
@@ -212,6 +212,10 @@ jobs:
         run: npm install -g @microsoft/rush@5.112.1
       - name: Install dependencies
         run: rush install --max-install-attempts 3
+      - uses: browser-actions/setup-chrome@v1
```

Comment:
I'm wondering if this can be used in `get_matrix` job (or something similar and be added to output, so later we will need to update it in 1 place?

Reply:
Each job runs on a different agent allocated/isolated space, but this is a good idea. I'm going to check in as is to unblock other PRs and then try this out